### PR TITLE
Fix: Header/Narrow click in MessageListWeb

### PR DESCRIPTION
**Pull request by kunall17**

### CODE REVIEW

File: src/message/html/js.js [hunk 1]
```diff
@@ -63,28 +63,20 @@ window.addEventListener('scroll', () => {
...
-
   if (e.target.matches('.avatar-img')) {
     sendMessage({
       type: 'avatar',
       fromEmail: e.target.getAttribute('data-email'),
     });
+    return;
   }
 
-  if (e.target.matches('.header')) {
```
Comment (borisyankov) [inline src/message/html/js.js]: That is definitely a correct way to handle this. If some of the html rendered does not have the 'header' class, then this is where the change should happen!

File: src/message/html/js.js [hunk 2]
```diff
@@ -63,28 +63,20 @@ window.addEventListener('scroll', () => {
...
+  if (e.target.className === 'title-text' || e.target.className === 'private-header header' || e.target.className === 'topic-header header' || e.target.className === 'stream-text') {
     sendMessage({
       type: 'narrow',
       narrow: e.target.getAttribute('data-narrow'),
-      id: e.target.getAttribute('data-id'),
     });
+    return;
```
Comment (borisyankov) [inline src/message/html/js.js]: This is working as expected. No need to add a `return` keyword.
Multiple handlers can intersect.

**Comment by kunall17** [comment]
@borisyankov Updated

**Comment by borisyankov** [comment]
Looks good.

**Pull request merged**